### PR TITLE
support umd build

### PR DESCRIPTION
**Pull request by kotarella1110**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -4,30 +4,58 @@ import typescript from 'rollup-plugin-typescript2';
...
-  input: 'src/index.tsx',
-  output: [
-    {
-      file: pkg.main,
-      format: 'cjs',
+export default [
+  {
+    input: 'src/index.tsx',
+    output: {
+      name: 'ReactHookFormDevtools',
```
Comment (kotarella1110) [inline rollup.config.js:14]: ReactHookFormDevtools vs ReactHookFormDevTools

Comment (bluebill1049) [reply rollup.config.js:14]: ReactHookFormDevTools please :)

Comment (kotarella1110) [reply rollup.config.js]: done 👍

Comment (bluebill1049) [reply rollup.config.js]: awesome! thanks @kotarella1110

**Pull request merged**